doc/index.rst

```diff
@@ -143,7 +143,7 @@ Official releases of Param are available on
 ``conda install -c ioam param``, ``pip install --user param``, or 
 ``pip install param``.
 
-The very latest changes can be obtained via ``conda install -c cball
+The very latest changes can be obtained via ``conda install -c pyviz
```

Comment:
Shouldn't it say /label/dev if it's the "very latest"?

Reply:
Yes, well spotted, that's a mistake.